### PR TITLE
Added oxygen page

### PR DESCRIPTION
**Pull request by SukiCai**
## Description
Added the page for Oxygen

## Status
- [x] Ready to go

### CODE REVIEW

File: elements/oxygen.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: Oxygen
+---
```
Comment (Feeelo) [inline elements/oxygen.md:4]: Page layout & format is correct

**Review by Feeelo — APPROVED** [review]

**Comment by Feeelo** [comment]
Checked file name, page layout & information about element is included

File: elements/oxygen.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+layout: page
+title: Oxygen
```
Comment (codyingyao) [inline elements/oxygen.md:3]: formatting looks good~

**Review by codyingyao — COMMENTED** [review]
Looks good!

**Review by ellemaclennan — COMMENTED** [review]
Looks good!

**Pull request merged**